src/hooks/useListing.js

```diff
@@ -0,0 +1,66 @@
+import {useInfiniteQuery, useQuery} from 'react-query';
+import axios from 'axios';
+import {apiEndpoint, marketContractAddress, zoombiesContractAddress} from "../constants";
+import {getTokenSymbol} from "../utils/auction";
+
+export const getAuctionListings = async (marketContract, zoombiesContract, filters, sorting, page) => {
+  // console.log({filters, sorting})
+
+  const getSortType = () => {
+    // switch(sorting.field) {
+    //   case 'auctionEnd':
+    //     return 'END_TIME'
+    //   case 'minPrice':
+    //     return 'MIN_PRICE'
+    //   case 'highestBid':
+    //     return 'HIGHEST_BID'
+    //   case '':
+    //     return null
+    //   default:
+    //     throw new Error(`Unhandled sort type: ${sorting.field}`)
+    // }
+    return sorting.field
+  }
+
+  const params = new URLSearchParams({
+    cardOrigin: filters.cardType,
+    saleToken: filters.token,
+    cardRarity: filters.rarity,
+    search: filters.keyword,
+    sortBy: getSortType() ?? '',
+    offset: page * 5,
+    limit: '5'
+  })
+
+  const listings = await axios.get(`https://api.zoombies.world/listings?${params.toString()}`)
```

Comment:
nit, since you already extracted api endpoint to constant.js, can you use the variable from there?

Reply:
define limit as constant value?

Reply:
What I meant was, maybe like `await axios.get('${endpoint}/listings${params.toString()}'`

Reply:
I've fixed this.